docs/docker-stack/docker-images-recipes/hadoop.Dockerfile

```
@@ -24,7 +26,7 @@ USER 0
 RUN mkdir -pv /usr/share/man/man1 \
     && mkdir -pv /usr/share/man/man7 \
     && curl -fsSL https://adoptopenjdk.jfrog.io/adoptopenjdk/api/gpg/key/public | apt-key add - \
-    && echo 'deb https://adoptopenjdk.jfrog.io/adoptopenjdk/deb/ buster main' > \
+    && echo "deb https://adoptopenjdk.jfrog.io/adoptopenjdk/deb/ ${DEBIAN_VERSION} main" > \
```

```suggestion
    && echo "deb https://adoptopenjdk.jfrog.io/adoptopenjdk/deb/ $(lsb_release -cs) main" > \
```
We don't need to pass the edition name as an environment variable, but we can check it with `lsb_release -c` command.

---

Right! good idea!